packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -175,6 +183,14 @@ class Xcode {
       return minorVersion >= kXcodeRequiredVersionMinor;
     return false;
   }
+
+  Future<RunResult> cc(List<String> args) {
```

Comment:
This seems unused?

Reply:
Agh - nice catch. Yeah it's in #16974 which has been waiting for tip-of-tree to go green before I can land it.